test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -1561,6 +1561,8 @@ def chown(self, hostname=None, path=None, uid=None, gid=None, sudo=False,
                            runas=runas, level=level)
         if ret['rc'] == 0:
             if gid is not None:
+                if runas is None:
+                    runas = _u
```

Comment:
chown has already happened before control reaches here. This check it to set runas as the user who owns the file so that it can also change its group.